app/models/miq_report/formats.rb

```diff
@@ -19,17 +19,17 @@ def self.available_formats_for(column, suffix, datatype)
     end
   end
 
-  def self.default_format_for_path(path, dt)
-    col = path.split('-').last.to_sym
-    sfx = col.to_s.split('__').last.try(:to_sym)
-    default_format_for(col, sfx, dt)
-  end
-
-  def self.default_format_for(column, suffix, datatype)
```

Comment:
Is this not use anywhere else?

Reply:
Doesn't look like it! I preferred to inline it rather than change the signature in this case

Reply:
referenced only in manageiq proper:

app/models/miq_report/formats.rb
app/models/miq_report/formatting.rb
app/models/miq_report.rb
spec/models/miq_report/formats_spec.rb

Reply:
@kbrock are you sure those are all hits, and not hitting `default_format_for_path`?

Reply:
For reference (after my change):

```sh
tim@tim-thinkpad:~/src/manageiq$ ag "default_format_for"
spec/models/miq_report/formats_spec.rb
20:  describe '.default_format_for_path' do
32:          subj = described_class.default_format_for_path("ChargebackVm-#{name}", datatype.first)
48:      expect(described_class.default_format_for_path("ContainerVolume-capacity", :text)).to be_nil
49:      expect(described_class.default_format_for_path("PersistentVolume-capacity", :text)).to be_nil
50:      expect(described_class.default_format_for_path("ContainerVolumeKubernetes-capacity", :text)).to be_nil

app/models/miq_report.rb
134:      :default_format    => Formats.default_format_for_path(path, data_type),

app/models/miq_report/formatting.rb
15:    format_name ||= MiqReport::Formats.default_format_for_path(col, nil)

app/models/miq_report/formats.rb
22:  def self.default_format_for_path(path, datatype)
36:    format = FORMATS[default_format_for_path(path, datatype)]
```

Reply:
No hits in the classic UI also, which is the only other place I might expect to find it